dev_guide/builds.adoc

```diff
@@ -1362,9 +1366,18 @@ git:
     name: "<committer name>"
     email: "<committer e-mail>"
   message: "<commit message>"
+env:
+   - name: "<variable name>"
+     value: "<variable value>"
+
 ----
 ====
 
+Similar to the link:#buildconfig-environment[*BuildConfig* environment] variables, the environment variables
+defined in the above payload will be made available to your build.  When variable names in the above payload
+collide with the names of the link:#buildconfig-environment[*BuildConfig* environment] variables, the variables
+in the payload will take precedence.
```

Comment:
i'd add a sentence here about "by default environment variables passed via webhook are ignored, you must set allowEnv to true on the webhook definition to enable this behavior"